src/services/schedule.ts

```diff
@@ -116,7 +116,8 @@ export class ScheduleService implements IScheduleService {
     }
 
     if (transaction.status !== prevStatus && transaction.status === Status.Pending) {
-      send(scheduled, 'scheduled');
+      // Don't send scheduled emails for Strategy items.
+      dontSendScheduleEmail(scheduled);
```

Comment:
I gave you an example of a good name yesterday and reminded you today and yet here it is, wrong and misleading again.